packages/firebase/src/model/notification/handlers/sendNotification.ts

```diff
@@ -0,0 +1,62 @@
+import { FastifyReply } from "fastify";
+import { SessionRequest } from "supertokens-node/framework/fastify";
+
+import { sendPushNotification } from "../../../lib";
+import DeviceService from "../../userDevice/service";
+
+import type { Message, TestNotificationInput } from "../../../types";
+
+const testPushNotification = async (
+  request: SessionRequest,
+  reply: FastifyReply
+) => {
+  const userId = request.session?.getUserId();
+
+  if (userId) {
```

Comment:
I think it is better to check if if (!userId) so that we do not have use else statement.